mobileapp/www/js/configuration.js

```diff
@@ -7,5 +7,7 @@
   app.host = 'kloakstaging.crypton.io';
```

Comment:
I forgot to mention that this file - should not be checked into the repo, rather, we will create a pre-build hook that will create it for us. The main take-away is that we want to keep configuration out of the code where possible.